openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -418,11 +418,16 @@ def create_gdeformer_collection(parent_collection: bpy.types.Collection):
     Args:
         parent_collection (bpy.types.Collection): Collection to create GDEFORMER col in
     """
-    # Create GDEFORMER collection
-    gdeformer_col = bpy.data.collections.new("GDEFORMER")
-    parent_collection.children.link(gdeformer_col)
+
+    # Create GDEFORMER Collection
+    gdeformer_col = None
     for obj in bpy.context.scene.objects:
         if obj.name.startswith("GDEFORM"):
+            # Check if the GDEFORMER collection exists to create it and link it to the parent collection in args only once
+            if not gdeformer_col:
+                gdeformer_col = bpy.data.collections.new("GDEFORMER")
+                parent_collection.children.link(gdeformer_col)
+            # Link GDEFORM objects to the GDEFORMER collection
```

Comment:
In fact this one is a bit incorrect. In this case adding one is not mandatory, but this would be appropriate:
```suggestion
            # Link object to GDEFORMER collection
```